api/CanvasRenderingContext2D.json

```diff
@@ -1581,10 +1581,10 @@
                 "version_added": null
               },
               "opera": {
-                "version_added": null
+                "version_added": "46"
```

Comment:
This is changing the "SVGImageElement_source_image" entry, but it looks like it's just doing mirroring. We don't generate tests for this entry.